job-dsls/jobs/downstream_pr_jobs.groovy

```diff
@@ -10,7 +10,7 @@ def final DEFAULTS = [
         timeoutMins            : 300,
         label                  : "rhel7 && mem24g",
         upstreamMvnArgs        : "-B -e -T1C -DskipTests -Dgwt.compiler.skip=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Drevapi.skip=true -Dmaven.javadoc.skip=true clean install",
```

Comment:
No change here?

Reply:
@triceo see my 2nd comment. I left this in intentionally, because those maven comman lines  have `-DskipTests`, so the test stability is not affected, and we can still take advantage of speedup caused by building in parallel. If you're aware of other potential issues caused by leaving that in, let me know.